compiler/IREmitter/Payload/patches/string.c

```diff
@@ -8,3 +8,11 @@ VALUE sorbet_vm_fstring_new(const char *ptr, long len) {
 VALUE sorbet_vm_str_uplus(VALUE str) {
     return str_uplus(str);
 }
+
+VALUE (*sorbet_rb_str_to_s_func(void))(VALUE) {
+    return rb_str_to_s;
+}
+
+VALUE sorbet_rb_str_to_s(VALUE str) {
+    return rb_str_to_s(str);
+}
```

Comment:
Why do we have both of these?  Can't we just say in the intrinsic that we expect the cached function to be `rb_str_to_s`?

(And then if we have that, I suppose we don't need the `KnownFunction` bits?  At least not for this particular method...)

Reply:
`rb_str_to_s` is `static` in string.c, which is what led to all of this. The entry in the inline cache will be `rb_str_to_s`, not `sorbet_rb_str_to_s`, so there's some hoops to jump through.

Reply:
Oh, of course.  Sorry, I thought these were in the codegen payload; this makes more sense now.